tools/dockerfile/grpc_artifact_manylinux2014_x86/Dockerfile

```diff
@@ -32,16 +32,19 @@ RUN curl -sSL https://rvm.io/pkuczynski.asc | gpg --import -
 # Use "--insecure" to avoid cert expiration error
 RUN curl -sSL --insecure https://get.rvm.io | bash -s stable
 
-# Install Ruby 2.6
-RUN /bin/bash -l -c "rvm install ruby-2.6 --32"
-RUN /bin/bash -l -c "rvm use --default ruby-2.6"
+# Install Ruby 3.0
```

Comment:
LGTM since this in itself is fine, but:

1) We haven't supported ruby 2.6 in a long time. The fact this docker image was using ruby 2.6 makes me think that we weren't actually using ruby with this docker image. We might consider removing ruby entirely.

2) We'll actually drop support for ruby 3.0 in a ~month so we *could* pick a more recent version (anything 3.1 to 3.4)